File: 02. C++/16. Funcion recursiva/Función recursiva.cpp
#include "iostream"
#include "string"
#include "stdlib.h"

using namespace std;

void funcion(int x){
        
    cout << x <<endl;
    
    if(x > 0){
        funcion(x - 1);
    }
    
}

int main(){

	funcion(10);
        
	system("pause");
	return EXIT_SUCCESS;
	
}

File: 01. JAVA/16. Funcion recursiva/Función recursiva.java
package prueba;

public class Prueba {
    
    static void funcion(int x){
        
        System.out.println(x);
        
        if(x > 0){
            funcion(x - 1);
        }
        
    }
    
    public static void main(String[] args) {

        funcion(10);
        
    }
    
}